### PR TITLE
Add support for LooksRare Seaport orders

### PR DESCRIPTION
**Pull request by tv3636**

### CODE REVIEW

File: src/jobs/looksrare-v2-sync/utils.ts [hunk 1]
```diff
@@ -188,3 +189,187 @@ export const fetchOrders = async (
...
+
+      const orders: LooksRareSeaportOrder[] = response.data.data;
+      const parsedOrders: {
+        kind: "seaport-v1.4" | "seaport-v1.5";
+        data: Sdk.SeaportBase.Types.OrderComponents;
+      }[] = [];
+
+      const values: any[] = [];
+      const handleOrder = async (order: LooksRareSeaportOrder) => {
+        order.protocol_address = Sdk.SeaportV15.Addresses.Exchange[config.chainId];
```
Comment (tv3636) [inline src/jobs/looksrare-v2-sync/utils.ts:256]: @georgeroman I didn't see the protocol address included in the LooksRare API response so I'm making an assumption here that it's v1.5

File: src/jobs/looksrare-v2-sync/utils.ts [hunk 2]
```diff
@@ -188,3 +189,187 @@ export const fetchOrders = async (
...
+            data: parsed.order.params as any,
+          });
+        }
+
+        values.push({
+          hash: order.hash.toLowerCase(),
+          target:
+            parsed?.order.getInfo()?.contract.toLowerCase() ||
+            order.protocol_data.parameters.offer[0].token.toLowerCase(),
+          maker: order.protocol_data.parameters.offerer.toLowerCase(),
```
Comment (tv3636) [inline src/jobs/looksrare-v2-sync/utils.ts:270]: @georgeroman Not sure if this is valid, I assume offerer is equivalent to maker but didn't see `maker` anywhere explicitly

File: src/jobs/looksrare-v2-sync/utils.ts (hunk 1 above)
Comment (georgeroman) [reply src/jobs/looksrare-v2-sync/utils.ts:256]: Yes, for sure they're using v1.5

File: src/jobs/looksrare-v2-sync/utils.ts (hunk 2 above)
Comment (georgeroman) [reply src/jobs/looksrare-v2-sync/utils.ts:270]: That's right, offerer = maker

**Pull request merged**